faq.html

```diff
@@ -219,11 +219,11 @@ <h5><b>Q. My question is not listed here / I am still unclear about a few things
 
           <ul class="list-inline">
 
-            <li><img src="img/fb.png" alt="image"></li>
+            <li><a href="https://www.facebook.com/PyConIndia" target="_blank"><img src="img/fb.png" alt="image"></a></li>
 
-            <li><img src="img/twitt.png" alt="image"></li>
+            <li><a href="https://twitter.com/pyconindia" target="_blank"><img src="img/twitt.png" alt="image"></a></li>
 
-          <ul>
```

Comment:
Can you please add mailing list button as in index.html page?

Reply:
done.